ansible/playbook.yml

```diff
@@ -63,14 +45,31 @@
       tags:
         - install-supabase-internal
 
-    - name: Clean out build dependencies
-      import_tasks: tasks/clean-build-dependencies.yml
-
     - name: Adjust APT update intervals
       copy:
         src: files/apt_periodic
         dest: /etc/apt/apt.conf.d/10periodic
 
+    - name: Transfer init SQL files
+      copy:
+        src: files/{{ item.source }}
+        dest: /tmp/{{ item.dest }}
+      loop: "{{ sql_files }}"
+
+    - name: Execute init SQL files
+      become: yes
+      become_user: postgres
+      shell:
+        cmd: /usr/lib/postgresql/bin/psql -f /tmp/{{ item.dest }}
+      loop: "{{ sql_files }}"
+      when: not ebssurrogate_mode
```

Comment:
Not sure I follow why we're not executing these files?

Reply:
Inside `ansible-chroot` I'm unable to start `postgresql`  with systemd. This looks like known ansible issue. So I skipped these init part, thinking "we are not starting postgresql during first boot, may be this can be handled with the script which starts  postgresql ".  

Now found a work-around for this, instead of using `systemd` updated PR uses `pg_ctl` to start this process.